CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This log is intended to keep track of package changes, including
 but not limited to API changes and file location changes. Minor behavioral
 changes may not be included if they are not expected to break existing code.
 
+## 0.3.12 (2023-03-14)
+
+- Add support for template groups including new message type "init-template-group"
```

Comment:
Since this is customer facing, we probably don't need to mention the 'init-template-group' message, they won't ever need to interact with it